src/Microsoft.Extensions.Options/Microsoft.Extensions.Options.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.Extensions.Primitives" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="System.ComponentModel" Version="$(CoreFxVersion)" />
+    <PackageReference Include="System.Collections.Concurrent" Version="$(CoreFxVersion)" />
```

Comment:
Oops yeah I tried adding this before realizing that I just needed to bump it up to 1.1 for it to show up, I'll remove